IlmBase/ImathTest/testFrustum.cpp

```diff
@@ -57,17 +57,17 @@ testFrustumPlanes  (IMATH_INTERNAL_NAMESPACE::Frustumf &frustum)
     IMATH_INTERNAL_NAMESPACE::V3f o (0.0f, 0.0f, 0.0f);
     float eps = 5.0e-4;
 
-    for (float xRo = 0.0f; xRo < 360.0f; xRo += 100.0f)
+    for (auto xRo : {0.0f, 100.0f, 200.0f})
```

Comment:
I guess you need auto instead of float to use this snazzy iteration syntax? It is cool...

Reply:
No you can still explicitly declare the type when using range-based for, so it's fine to declare 'float' here.  Do we have a coding standard wrt to usage of auto? For UE4, our standard is that we always explicitly declare the type. The only exceptions where we use 'auto' are when declaring a variable set to a lambda, verbose iterator types, or in template code where the type cannot easily be determined. The advantage of auto's is preventing implicit conversion. But this is only beneficial when auto is used ubiquitously. Here being explicit by declaring the type float seems preferable.

Reply:
I don't think we've taken a stance on auto, but when we address it I would prefer the UE4 convention you describe.

Reply:
Sounds good, I leave it to Cary whether he wants to change this one or not.

Reply:
My vote is also for using the explicit type when it's something that is known and can be written compactly.

`float x` is better than `auto x`

But I totally respect `auto x` when it can replace an ugly mouthful like `std:vector<std::pair<FooType,std::map<int>&>::const_iterator`.